docs/UsersGuide/source/BackgroundInfo/Components.rst

```diff
@@ -6,21 +6,21 @@ SRW Application Components
 
 The SRW Application assembles a variety of components, including:
 
-* Pre-processor Utilities & Initial Conditions
+* UFS Utilities
 * UFS Weather Forecast Model
 * Unified Post Processor
 * METplus Verification Suite
 * Unified Workflow Tools
-* Build System and Workflow
+* Build system and workflow
```

Comment:
Yeah, the proper names should be capitalized, but I suppose "verification suite" isn't part of the METplus name, for example. Is UFS_UTILS technically short for UFS Utilities, or is it more that UFS_UTILS is the name, and UFS utilities is a description of what it is?

Reply:
I was wondering about this too.  Then I figured @gspetro-NOAA is doing what she said --  if it's part of the official name, then capitalize, otherwise don't.  The difficulty then is to figure out what is part of the official name and what isn't.  For UFS_UTILS, I'd say if you're going to capitalize "UFS Weather Model" because that's the official name, then also capitalize "utilities" in "UFS Utilities".  For METplus, I haven"Unified Workflow", I think the "Tools" is part of the repo description, so capitalize that. lower case.  But either way it's not a big deal, and the names/descriptive phrases are going to change over time anyway.

Reply:
@gsketefian Your reply got a little garbled (maybe a mistaken copy-paste mid-sentence?). Not sure what you were suggesting re: METplus.

Reply:
@gspetro-NOAA Sorry about that.  Just updated my comments above.

Also, at least in my browser, the first bullet point now says "UFS Utilitiesstatistical driver", which seems to be a typo.